src/core/patch.rkt

```diff
@@ -34,7 +34,8 @@
                               #;(log ,log-x ,exp-x))))
 
 (define (taylor-alt altn)
-  (define expr (alt-expr altn))
+  (define spec (alt-expr altn))
```

Comment:
Why do we call this `spec` if it is a prog?

Reply:
Fixed